qadence/backends/pyqtorch/convert_ops.py

```diff
@@ -417,13 +422,74 @@ def forward(
         state: Tensor,
         values: dict[str, Tensor],
     ) -> Tensor:
-        return apply_operator(
-            state,
-            self.unitary(values),
-            self.qubit_support,
-            self.n_qubits,
-            self.batch_size,
-        )
+        if self.block.generator.is_time_dependent:  # type: ignore [union-attr]
+
+            def Ht(t: Tensor | float) -> Tensor:
+                # values dict has to change with new value of t
+                # initial value of a feature parameter inside generator block
+                # has to be inferred here
+                new_vals = {}
+                for str_expr, val in values.items():
+                    expr = sympy.sympify(str_expr)
+                    t_symb = sympy.Symbol("t")
+                    free_symbols = expr.free_symbols
+                    if t_symb in free_symbols:
+                        # expression has time dependence
+                        if len(free_symbols) == 2:
+                            # expression has a single additional feature parameter
+                            feat_symb = free_symbols.difference(set([t_symb])).pop()
```

Comment:
Because I can't know the original value of some feature parameter sitting inside my time-dependent generator. Again, due to embedding stuff. So I have to solve for original value of that feature parameter.